prototype_source/flight_recorder_tutorial.rst

```diff
@@ -169,9 +174,28 @@ To run the convenience script, follow these steps:
 
 2. To run the script, use this command:
 
-.. code:: python
+.. code:: shell
+
+  python fr_trace.py <dump dir containing trace files> [-o <output file>]
+
+Or if you install PyTorch nightly build or build from scratch (with ``USE_DISTRIBUTED=1``), you can directly use the following command:
+
+.. code:: shell
+
+  torchfrtrace <dump dir containing trace files> [-o <output file>]
+
+
+For now, we support two modes for the analyzer script, one is to let the script to apply some heuristics to the parsed flight recorder
+dumps to generate a report on potential culprit for the timeout/hang; the other one is to just print out raw dumps. For the latter, by
+default the script prints for all ranks and all ProcessGroups(PGs), and this can be narrowed down to certain ranks and PGs. Example
```

Comment:
```suggestion
the script, by default, prints information for all ranks and all ``ProcessGroups``(PGs). This can be narrowed down to certain ranks and PGs. An example
```